agent/cmd/checkPermission.go

```diff
@@ -204,7 +244,15 @@ func getCertInfo(serialNumber string, keyID string, api string) (CertInfo, error
 	}
 
 	// Get certificate from API
-	resp, err := netClient.Get(fmt.Sprintf("%s/certificates/%s?key_id=%s", api, url.QueryEscape(serialNumber), url.QueryEscape(keyID)))
+	resp, err := netClient.Get(
+		fmt.Sprintf("%s/certificates/%s?key_fingerprint=%s&key_id=%s&certificate=%s&certificate_type=%s",
+			api,
+			url.QueryEscape(serialNumber),
+			url.QueryEscape(keyID),
```

Comment:
Based on how you are building this URL, maybe `keyFingerprint` comes before `keyID` here?

Reply:
Great appointment! Corrected!